CODIGO/Declares.bas

```diff
@@ -34,7 +34,7 @@ Attribute VB_Name = "Mod_Declaraciones"
 Option Explicit
 
 #If False Then
-    Dim nombre As String
```

Comment:
Can you restore this in your VB file so all the variables will be restored to their original case?